### ISSUE TITLE
Update: Use Tailwind CSS instead of separate CSS files

### ISSUE THREAD

**Issue by Sambit-Mondal** [vishalverma9572/QuizMaster#46]
## Reasons:

- If we use Tailwind CSS instead of separate CSS files, it'd make the app more light-weight, and would increase the speed of the application.
- It'd reduce the chances of error of other contributors contributing to this project, as they don't have to write the CSS from scratch.
- It'd be easy to understand too, as we don't have to find each element separately in the respective CSS files through their respective classnames.
- Since it is built on MERN Stack, having Tailwind CSS as the styling framework would be an industry level enhancement in this project.

**Comment by vishalverma9572** [comment]
@Sambit-Mondal , can you explain how you're planning to work on this? Are you going to restyle all the components similarly, or do you have a different approach?

**Comment by Sambit-Mondal** [comment]
@vishalverma9572 Yeah, I was thinking to restyle all the components similarly, as they are, but using Tailwind CSS. Will it be okay?

**Comment by vishalverma9572** [comment]
@Sambit-Mondal , yes, work on the components one by one. If there are no issues, create a PR, and after that, you can move on to other components. We follow a specific styling flow where we create a CSS file with the same name as the component, and for each style, we add .parent_div in that component's classes or IDs. Please keep this in mind.

**Comment by vishalverma9572** [comment]
@Sambit-Mondal , any updates?

**Comment by Sambit-Mondal** [comment]
@vishalverma9572 Actually, I didn't work in the previous week, as I was out of station for Durga Puja. I'll start working again by tomorrow, and will complete it and create a PR by the end of this week. 

Sorry for the delay and thanks for your patience! 😊
